lib/net.js

```diff
@@ -737,21 +738,21 @@ Socket.prototype._writeGeneric = function(writev, data, encoding, cb) {
   if (this.connecting) {
     this._pendingData = data;
     this._pendingEncoding = encoding;
-    this.once('connect', function connect() {
+    this.once('connect', () => {
```

Comment:
This seems an unnecessary change at this moment.

Reply:
In the notes, I mentioned that I made the change out of curiosity, because I had no idea why this statement worked in the first place. Why would we use a `function` call here over an arrow function? Won't the `function` call *not* retain `this`? Isn't this like... the exact use case we needed arrow functions for? I'm confused and needed help figuring this one out.

Reply:
Either is fine here. EventEmitter calls the function with the `this` being bound to the EventEmitter instance, so in this case it doesn't matter if we use an arrow function or not (in other cases this change would be correct). I just don't think we should change it unless we're gaining something by doing it.